templates/products/products-listing.php

```diff
@@ -290,7 +290,6 @@
 
                         </table>
                     </form>
-                </div>
```

Comment:
Why would you remove this closing `div` is unclear. Since, it closes the opening `div.dokan-dashboard-product-listing-wrapper`